interface/main/tabs/menu/menus/standard.json

```diff
@@ -1238,14 +1238,26 @@
             "label":"Weno Management",
             "menu_id":"adm0",
             "target":"adm",
-            "url":"/interface/weno/admin.php",
+            "url":"/interface/weno/facilities.php",
             "children":[],
             "requirement":0,
             "acl_req": [
               "admin",
               "super"
             ]
```

Comment:
should this entry only show if the weno globals is on?

Reply:
I can agree